tests/test/scheduler/test_function_client_server.cpp

```diff
@@ -53,32 +53,50 @@ class ClientServerFixture
 
 TEST_CASE_METHOD(ClientServerFixture, "Test sending MPI message", "[scheduler]")
 {
-    // Create an MPI world on this host and one on a "remote" host
-    std::string otherHost = "192.168.9.2";
+    auto& sch = faabric::scheduler::getScheduler();
+
+    // Force the scheduler to initialise a world in the remote host by setting
+    // a worldSize bigger than the slots available locally
```

Comment:
Again, very small nitpicky thing but having variable names in comments not ideal. Here it can just be "world size" rather than "worldSize"